### ISSUE TITLE
Introduce new configuration format

### ISSUE THREAD

**Issue by wata727** [wata727/tflint#191]
TFLint has focused on detecting errors until now, but I'll focus on more other problems (e.g. https://github.com/wata727/tflint/issues/178, https://github.com/wata727/tflint/issues/174) in the future.

For that, I'm planning to introduce the new configuration format like the following:

```hcl
config {
  // it also works
  ignore_rule = {
    aws_instance_invalid_type  = true
    aws_instance_previous_type = true
  }
}

// new format
rule "aws_instance_invalid_type" {
  enabled = false
}

rule "aws_instance_previous_type" {
  enabled = false
}
```

This allows you to introduce rules that are disabled by default. In addition, you can pass more detailed settings for each rule.
I have not completed the design yet, so it may change in the future. Feedbacks are welcome. Thanks!
